data/remnant/remnant jobs.txt

```diff
@@ -179,7 +179,7 @@ mission "Remnant: Bounty 4"
 			names "korath"
 			cargo 3
 			variant
-				"Korath Dredger (Hyperdrive)"
+				"Korath Dredger"
```

Comment:
No, it's just that the ship already had a jump drive anyway.  The (Hyperdrive) variant was just the Dredger equivalent of the Raider's (Ember) variant, denoting that it had a Hypedrive in addition to its Jump Drive.  It was misleading, and serves no purpose given all dredgers have hyperdrives now anyway, so I deleted it.

If you wanted to change the mission so that the dredger cannot jump, then you would need to use the (Crippled) variant, which some missions already do, but I don't want to go around *actually* changing Remnant missions in this PR.